README.md

```diff
@@ -146,6 +146,7 @@ In the documentation you will see examples using [object spread properties](http
     <li><a href="http://polished.js.org/docs/#rgbacolor">RgbaColor</a></li>
     <li><a href="http://polished.js.org/docs/#rgbcolor">RgbColor</a></li>
     <li><a href="http://polished.js.org/docs/#timingfunction">TimingFunction</a></li>
+    <li><a href="http://polished.js.org/docs/#getluminance">getLuminance</a></li>
```

Comment:
This is getting put in the wrong spot, due to it missing from `documentation.json`. If you add it, this *should* get inserted into the proper place in the readme and the documentation.